index.html

```diff
@@ -963,8 +963,7 @@ <h2>Methods</h2>
                 <li>If <var>amount</var> is larger than the value of
                 <var>stream</var>'s <a>[[\TargetReadBufferedAmount]]</a> slot,
                 <a>reject</a> <var>p</var> with a newly created <code>OperationError</code>.</li>
-                <li>Let <var>threshold</var> be the value of <var>stream</var>'s
-                <a>[[\TargetReadBufferedAmount]]</a> slot minus <var>amount</var>.</li>
+                <li>Let <var>threshold</var> be <var>amount</var>.</li>

```

Comment:
Don't really need a `threshold` variable; can just reference `amount` in the step below